README.md

```diff
@@ -224,29 +242,24 @@ cd che
 git checkout openshift-connector
 ```
 
-* Set some environment variables (or run `. ./scripts/setenv-for-build.sh`)
+* Set some environment variables:
+    * `UPSTREAM_CHE_PATH` points to the local directory where the official Che repo was checked out
+    and is *mandatory* for some scripts
 
 ```bash
-export CHE_IMAGE_REPO=rhche/che-server
-export CHE_IMAGE_TAG=nightly
```

Comment:
Why are you removing these? How a user is going set the name and tag of the Docker image?